db/src/excision.rs

```diff
@@ -127,3 +133,108 @@ pub(crate) fn excisions<'schema>(partition_map: &'schema PartitionMap, schema: &
         Ok(Some(excisions))
     }
 }
+
+pub(crate) fn enqueue_pending_excisions(conn: &rusqlite::Connection, schema: &Schema, tx_id: Entid, excisions: ExcisionMap) -> Result<()> {
```

Comment:
While this is called "enqueue", the function actually mutates state beyond just making a note of an excision-to-be-processed. Not in a permanent sense (we can always derive erased datoms if the `ensure_no_pending_excisions` step doesn't run, but it still does more than it lets on, I think.

Reply:
This is a fair comment.  Maybe `begin_excisions`?